webdriver-spec.html

```diff
@@ -3237,183 +3237,428 @@ <h2 id=element-displayedness>Element Displayedness</h2>
-                                    <!-- Should there be a recursion limit? -->
-                                    <li>a <code>Sequence &lt;Node> Nodes</code>, then return the result by recursively applying this algorithm to <code>result</code>. WebDriver implementations SHOULD limit the recursion depth.</li>
-                                    <!-- Should there be a limit? -->
-                                    <li>an object, then return the <code>object</code> created by recursively applying this algorithm to each property in <code>result</code>.</li>
-                                  </ol>
-                                </li>
-                              </ol>
-                            </li>
-                            <li>Set the command's response <a href="#response-value">value</a> to <code>value</code>.</li>
-                          </ol>
-                        </li>
-                        <li>Return the response.</li>
-                      </ol>
-                    </li>
-                  </ol>
-                </section>
-        </section>
-        <section>
-          <h2>Reporting Errors</h2>
-          <p></p>
-        </section>
+            </dd>
+
+            <dt>Otherwise:</dt>
+            <dd>
+              <ol>
+                <li><p>Let <var>result</var> be a
+                    new <code>Object</code> object.</p></li>
+                </ol>
+            </dd>
+          </dl>
+        </li>
+
+        <li><p>For each enumerable own property in <var>value</var>,
+        run the following substeps:</p>
+          <ol>
+            <li><p>Let <var>name</var> be the name of the property.</li>
+
+            <li><p>Let <var>source property value</var> be the result
+                of <a>getting a property</a> named <var>name</var>
+                from <var>value</var>. If doing so causes script to be
+                run, and that script throws an exception,
+                return <a>error</a> with code <a>javascript
+                error</a>.</li>
+
+            <li><p>Let <var>cloned property result</var> be the result of
+                calling the <var>clone algorithm</var> with
+                arguments <var>source property value</var>
+                and <var>seen</var>.</p></li>
+
+            <li><p>If <var>cloned property result</var> is
+                a <a>success</a>, <a>set a property</a>
+                of <a>result</a> with name <var>name</var> and value
+                equal to <var>cloned property result</var>'s data.</li>
+
+            <li><p>Otherwise, return <var>cloned property result</var>.</li>
+          </ol>
+        </li>
+  </ol>
+
+  <p>The steps for <dfn>extracting the script arguments from a request</dfn>,
+  are:</p>
+   <ol>
+      <li><p>If the <a>current browsing context</a> is <a>no longer
+          open</a>, return an <a>error</a> with code <a>no such
+          window</a>.</p></li>
```

Comment:
I don’t think this should be part of this definition, let’s put it on each command instead.